src/components/Button.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const Button = ({ name = 'Button' }) => <button type="button">{name}</button>;
+
+export default Button;
```

Comment:
## [ OPTIONAL ]
Please rename this prop to something like `buttonName` or something similar/intuitive, so that we can track it, because your code will add up in next milestones, many things will have names

## [REQUIRED]
Also enforce this NAME to be of `propType` STRING ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png))

(you can read on what's proptypes, it's a simple notion)